examples/stm32/xively_demo_mt/Src/system_stm32f4xx.c

```diff
@@ -1,21 +1,21 @@
 /**
   ******************************************************************************
-  * @file    system_stm32f4xx.c

```

Comment:
"I wonder if we should prepare example per each SDK version"  

Anyone else nervous about setting a precedent like this? Or is it just me?